assets/js/app.js

```diff
@@ -434,34 +434,38 @@ $(window).on("load", function () {
 //send button animation
 
 
-$(function submitAnimation () {
-const name=  document.querySelector("#name")
-const emailAdress=  document.querySelector("#email")
-const text=  document.querySelector("#textArea")
+$(function submitAnimation() {
+  const name = document.querySelector("#name")
+  const emailAdress = document.querySelector("#email")
+  const text = document.querySelector("#textArea")
+  const emailPattern = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
 
   $("#lnch").on("click", function () {
-     
-     if(name.value==""  ){
-      return
-     }
-   else if(emailAdress.value=="" ){
-  
-       return
+
+    if (name.value == "" || (/\d/.test(name.value))) {
+      alert('Please enter a valid name');
     }
-  
-  else  if(text.value==""){
-      return
+    else if (emailAdress.value == "" || !(emailPattern.test(emailAdress.value))) {
+      alert('Please enter a valid email');
```

Comment:
Could you add some comments to make the code more readable for future developments please?

For example:
- Checking if the email entree is valid in the form abc@x.com
-  Form Carry Endpoint Key
-  Send Button Animation
- etc

Reply:
Okay, I will add comments now